.buildkite/bin/update_version.sh

```diff
@@ -6,7 +6,7 @@ set -eu
 # $2 Package.swift file location
 # $3 README.md file location
 
-FILE_LOCATION="https://rokt-eng-us-west-2-mobile-sdk-artefacts.s3.amazonaws.com/ios/"$1"/Rokt_Widget.xcframework.zip"
+FILE_LOCATION="https://apps.rokt.com/msdk/ios/"$1"/Rokt_Widget.xcframework.zip"
```

Comment:
Need to update the regex `perl -pi -e "s/(?<=amazonaws.com\/ios\/)(.*)(?=\/Rokt)/$1/g" $2` on line 14 as well.
Changing `amazonaws.com` to `msdk` should probably work

Reply:
😨 Good catch. Fixed now.

Reply:
ty 🙏